tests/main.nf.test

```diff
@@ -235,4 +237,31 @@ nextflow_pipeline {
         }
 
     }
-}
\ No newline at end of file
+
+    test("Private study reads - this one should fail") {
+
+        tag "samplesheet"
+        tag "private"
+
+        when {
+            params {
+                outdir = "tests/results"
+                assembler = "spades"
+                bwamem2_reference_genomes_folder = "${projectDir}/tests/human_phix/bwa2mem"
```

Comment:
bwamem2+blast paths can be removed (already defined in test.config)
Those in "Samplesheet spades - retries" are residues I forgot to delete

Reply:
Perfect.. I'll leave this change for another PR I have in the works